README.rst

```diff
@@ -254,16 +246,14 @@ You can find a list of the available endpoints and example input/output in the
 'REST API' tab in this NApp's webpage in the `Kytos NApps Server
```

Comment:
@ajoaoff it's no directly related to this PR, but this part here regarding the docs of the rest api, maybe we could link it directly to the spec file `./openapi.yml`? and then later on, potentially we could try to render on GH pages, if ever needed, when someone asks for it? wdyt?

Reply:
They are already rendered (at least for some NApps) in GH pages. See https://kytos-ng.github.io/api/api.html

Reply:
really neat @ajoaoff, thanks for the info.